forc-plugins/forc-tx/src/lib.rs

```diff
@@ -621,20 +635,19 @@ impl TryFrom<Create> for fuel_tx::Create {
             .into_iter()
             .map(|s| fuel_tx::Witness::from(s.as_bytes()))
             .collect();
+
+        let mut policies = Policies::default().with_maturity(create.maturity.maturity.into());
+        policies.set(PolicyType::GasPrice, create.gas.price);
```

Comment:
```suggestion
        let mut policies = Policies::default().with_maturity(create.maturity.maturity.into()).with_gas_price( create.gas.price);
```

Also, we don't need to set `maturity` if it is zero.